src/sections/Community/Handbook/mentorships.js

```diff
@@ -32,6 +32,10 @@ const ConductWrapper = styled.div`
           display: block;
           margin-left: auto;
           margin-right: auto;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
```

Comment:
Why go so far as to just center an image? Moreover, this creates issues in mobile view.
Use `align-self: center` on the `Col` component having the img i.e, use the `id` attribute of the component to apply CSS to all the `Col` having the img as a child in this page.

Also, after doing so remove the `padding-top` CSS applied to the img at line: 29

Reply:
Thanks for the suggestions!